config/emailService.js

```diff
@@ -1,13 +1,16 @@
 const nodemailer = require("nodemailer");
+var htmlToText = require("nodemailer-html-to-text").htmlToText;
 
-let { EMAIL_USER, EMAIL_PASS } = require("./index");
+let { EMAIL_USER, EMAIL_PASS, EMAIL_HOST, SENDER_EMAIL } = require("./index");
 let { getMailTemplate } = require("../utility/emailTemplates");
 const { logger } = require("../utility/helpers");
 
 const transporter = nodemailer.createTransport({
-	service: "gmail",
 	type: "SMTP",
-	host: "smtp.gmail.com",
+	host: EMAIL_HOST,
+	secure: true,
+	debug: true,
+	port: 465,
 	auth: {
 		user: EMAIL_USER,
```

Comment:
since our email service is replaced, is this auth parameter required?

Reply:
Yes, any email service require auth